src/components/Flag/Flag.tsx

```diff
@@ -2,59 +2,87 @@ import React from 'react'
 import unknownFlag from '@assets/unknown-flag.png'
```

Comment:
Could you please create an example in a new storybook component plz ?

Reply:
I updated the API Reference section in the docs. Is that what you mean?

Reply:
Hmm, i'm lost, I have no idea what to add and where. This is the storybook file you are referencing?
`src/index.stories.tsx`?

But what is a "Template"? Is it a `StoryFn` like "Primary"? What do you want me to put there? I also don't get the idea of that Primary page to be honest.

Reply:
Ok no worries, I'm taking care of it

Reply:
What other CDN you have for example plz?

Reply:
Within our company I already created a CDN before for hosting other assets. Im going to use this CDN also for the flags, but it's meant to be to communicated or used for the outside world unfortunately. I assume you want to use an other CDN as an example in the storybook?

About the `flagSize`, as you know I didn't touch that part, but i agree that it's not very UI friendly. Could be another improvement :) but if people are using it, it would be a breaking change :(

Reply:
I agree that is would be (a bit) more easy for devs, but also it's not that hard at the moment right?

Not sure what the pro's and con's are with the `<picture />` tag. From what I read (w3schools) is that the picture tag especially useful is for responsive designs. And that is not the case for the flags, so yes could be refactored, but don't have a strong opinion.